### PR TITLE
Get correct model output for hydrology

### PR DESCRIPTION
**Pull request by mmcfarland**
The hydrology output from this port of GWLF-E was incorrect.  This PR gets the values to align, with some minor rounding differences, to the original vb6 implementation.   Fixes include:

- Fix indentation for monthly loop that was included in daily loop
- Don't use `-=` syntax, which has subtly different order of operations than the straight vb syntax
- Include missing conditional and assignment

These changes align all output fields (except AvPrecip which was already correct) but may not guarantee that the sedimentation values yet to be ported are correct.

Connects https://github.com/WikiWatershed/gwlf-e/issues/38

#### To test:
Ideally one would check my changes against the VB code, but it is sufficient to run the model on `/testinput_4.gms` and then use the `compare_output` script, as referenced in `README` to check against the known values from the compiled VB model.

### CODE REVIEW

File: gwlfe/CalcCnErosRunoffSed.py [hunk 1]
```diff
@@ -163,8 +163,8 @@ def CalcCN(z, i, Y, j):
             if z.Water >= 0.2 * z.CNumPervReten:
                 z.QrunP[l] = (z.Water - 0.2 * z.CNumPervReten) ** 2 / (z.Water + 0.8 * z.CNumPervReten)
 
-        # TODO: Should 11 be NRur + 1? What is this trying to do?
-        lu = l - 11
+        # TODO: What is this trying to do?
+        lu = l - 10
```
Comment (kdeloach) [inline gwlfe/CalcCnErosRunoffSed.py]: Consider using `z.NRur` instead of 10.

File: gwlfe/gwlfe.py [hunk 2]
```diff
@@ -116,7 +119,7 @@ def run(z):
 
                 # UPDATE ANTECEDENT RAIN+MELT CONDITION
                 # Subtract AMC5 by the sum of AntMoist (day 5) and Water
-                z.AMC5 -= z.AntMoist[4] + z.Water
+                z.AMC5 = z.AMC5 - z.AntMoist[4] + z.Water
```
Comment (kdeloach) [inline gwlfe/gwlfe.py]: Huge mistake on my part. Nice catch.

File: gwlfe/gwlfe.py [hunk 3]
```diff
@@ -190,6 +193,8 @@ def run(z):
                 # CALCULATE TOTALS
                 z.Precipitation[Y, i] = z.Precipitation[Y, i] + z.Prec[Y, i, j]
                 z.Evapotrans[Y, i] = z.Evapotrans[Y, i] + z.ET
+
+                #print("%s %s" %(j, z.Evapotrans[Y, i]))
```
Comment (caseypt) [inline gwlfe/gwlfe.py]: Left over debug statement?

**Comment by caseypt** [comment]
+1. Tested. Nice work. The output is much closer.

File: gwlfe/CalcCnErosRunoffSed.py (hunk 1 above)
Comment (mmcfarland) [inline gwlfe/CalcCnErosRunoffSed.py]: I asked for clarification on `11` and if `NRur` was appropriate.  The response was there might be a bug in the original, so I'm going to hold off until I hear back. :confused:

File: gwlfe/gwlfe.py (hunk 3 above)
Comment (mmcfarland) [inline gwlfe/gwlfe.py]: Yep, will remove, thanks.

**Comment by mmcfarland** [comment]
Added fixup commits for extra print statement and z.NRur 
@kdeloach, looks ok?

File: gwlfe/CalcCnErosRunoffSed.py (hunk 1 above)
Comment (kdeloach) [inline gwlfe/CalcCnErosRunoffSed.py]: I think your code is correct. 

`ISRR` and `ISRA` should have a length of `6` (which is `NUrb`, https://github.com/WikiWatershed/gwlf-e/blob/develop/gwlfe/parser.py#L995). And this loop iterates from `NRur` to `NLU` which should be 10 to 16

Comment (mmcfarland) [inline gwlfe/CalcCnErosRunoffSed.py]: Great, that's the intention that Ken supplied in our conversation.

**Comment by kdeloach** [comment]
:+1:

**Pull request merged**